python/feathub/processors/flink/table_builder/join_utils.py

```diff
@@ -71,13 +71,15 @@ def __init__(
 
     @staticmethod
     def from_table_descriptor_and_field_name(
-        table_descriptor: TableDescriptor, field_name: str
+        table_descriptor: TableDescriptor, field_name: str, set_default_value: bool
```

Comment:
Could you double check whether the python doc of `valid_time_interval` and `default_value` is correct?

It is not intuitive to understand the meaning of `set_default_value` here. Can you explain its semantics?

Would it be more intuitive to get the config from the input TableDescriptor and use this config to determine the expected behavior?